src/screenfull.js

```diff
@@ -1,3 +1,8 @@
+/*!
+* screenfull
+* v3.0.0 - 2015-11-24
+* (c) Sindre Sorhus; MIT License
+*/
```

Comment:
well, I saw Zepto add license in src file, so I did that. Actually I don't know what's the difference between them.